src/open_inwoner/scss/components/Typography/H3.scss

```diff
@@ -1,5 +1,4 @@
-@import '~@utrecht/heading-3-css/dist/index.css';
-@import '~microscope-sass/lib/responsive';
+@use '@utrecht/heading-3-css/dist/index.css';
```

Comment:
We still have a lot of vanilla CSS media queries instead of microscope-sass so I'm wondering wether we should fully use microscope-sass or not at. all.